packages/lambdas/src/cw-link-patients.ts

```diff
@@ -66,7 +66,7 @@ export const handler = Sentry.AWSLambda.wrapHandler(async (event: SQSEvent) => {
 
     console.log(`Done.`);
   }
-  console.log(`Total time: ${Date.now() - startedAt}ms`);
+  console.log(`Total time: ${Date.now() - startedAt} ms`);
```

Comment:
Updated EC's script so its easier to copy the duration w/ double click on terminal, replaced others as well